src/iocore/cache/test/storage.config

```diff
@@ -1,2 +1,2 @@
-var/trafficserver 32M
+var/trafficserver 256M
```

Comment:
If I set larger value than this, the test failed intermittently. I left the issue from this PR.
https://github.com/apache/trafficserver/issues/10648